packages/manager/apps/dedicated/client/app/network-security/traffic/traffic.component.js

```diff
@@ -4,6 +4,8 @@ import template from './traffic.html';
 export default {
   bindings: {
     getSubnet: '<',
+    ip: '<',
+    dateTime: '<',
```

Comment:
```suggestion
    ip: '<?',
    dateTime: '<?',
```
Because it's same component that used by both routing and only one have ip and dateTime in her resolve